src/__tests__/__snapshots__/role-helpers.js.snap

```diff
@@ -37,6 +37,28 @@ article ###################################
 [36m/>[39m
 
 
+command ###################################
+
+[36m<menuitem[39m
```

Comment:
Maybe we could use https://github.com/joaogranado/jest-serializer-ansi to make these snapshots better.

Reply:
Added jest-serilizer-ansi  

Also updated a couple other tests ([e452465](https://github.com/testing-library/dom-testing-library/pull/285/commits/e4524653e4b0212063d4d2e715cf99bdb479b7c2)) that had inline snapshots containing the same control characters.